app/lib/exceptions.rb

```diff
@@ -5,6 +5,7 @@ class Error < StandardError; end
   class NotPermittedError < Error; end
   class ValidationError < Error; end
   class RaceConditionError < Error; end
+  class WorkerError < Error; end
```

Comment:
Add exception for wrap exception in Sidekiq worker. (wrapped exceptions are raised to Sidekiq)